.gitignore

```diff
@@ -65,6 +65,7 @@ Doc/examples/tree1.nwk
 
 #Ignore LaTeX temp files, and compiled output
 Doc/*.aux
+Doc/*.gz
```

Comment:
Do you have ``Doc/*.gz`` files on your machine? I wonder why...

Reply:
Yeah, I do have a .gz file on my machine. I used TexMaker and when I select PDFLaTeX it creates Tutorial.synctex.gz file also.

Reply:
OK, this addition makes sense then 👍